ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh

```diff
@@ -67,6 +67,14 @@ EOF
 	  exit 1
   fi
 
+BYOC_KUBECONFIG="$SHARED_DIR"/kubeconfig
+if [[ -s "$BYOC_KUBECONFIG" ]]; then
+    yq e -i 'del(.clusters[].cluster.certificate-authority-data) | .clusters[].cluster.insecure-skip-tls-verify=true' $BYOC_KUBECONFIG
```

Comment:
what's the difference between `$KUBECONFIG` ([documented](https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables)) and the one you're using here?

just thinking if it's referring to the same one, this line might not be necessary, since it's already done [here](https://github.com/openshift/release/pull/39049/files#diff-275831e17826ca1203ee72a62de422ff411664d391d74122cf998655d773e602L49)

Reply:
I want to use the shared_dir for byoc and the other one for tests… i dont want to mix them